text/0115-mixed-shadow-mode.md

```diff
@@ -50,28 +51,30 @@ export default class extends LightningElement {
 ```
 
 This mode will result in the whole component subtree to operate in either native shadow mode or
-synthetic shadow mode. If the browser supports Shadow DOM the shadow mode will be native. If the
-browser does not support Shadow DOM (e.g., IE11) the shadow mode will be synthetic. The test to
+synthetic shadow mode. If the browser supports Shadow DOM, the shadow mode will be native. If the
+browser does not support Shadow DOM (e.g., IE11), the shadow mode will be synthetic. The test to
 determine Shadow DOM support will be to check whether `window.ShadowRoot` is defined.
 
 ### Opting out
 
-A component that is extending a component class that opted in to native shadow mode can opt out of
-native shadow mode by setting its own static `shadowSupportMode` property to `default`; however,
+A component that is extending a component class that opts in to native shadow mode can opt out of
+native shadow mode by setting its own static `shadowSupportMode` property to `reset`; however,
 this does not allow it to opt out of native shadow mode if any of its ancestor components set their
 static `shadowSupportMode` property to `any`.
 
-Setting `shadowSupportMode` to `default` is equivalent to not setting the property at all. A value
-of `default` results in the default behavior where the shadow mode of the component depends only
-on the inclusion of the `@lwc/synthetic-shadow` polyfill.
+Setting `shadowSupportMode` to `reset` acts to reset the shadow mode to the original default
+behavior where the mode depends only on the inclusion of the `@lwc/synthetic-shadow` polyfill:
+synthetic mode if the polyfill is present and native mode if it isn't.
 
-The only valid values are `default` and `any`. Any other value will result in a compiler error.
+The only valid values are `reset` and `any`. Any other value will result in a compiler error.

```

Comment:
@kevinv11n's model of a table with multiple dimensions is an interesting way to explain how `shadowSupportMode` influences the actual shadow DOM mode at runtime.

I am also wondering if a decision tree flow chart could make things easier for people to understand.

Reply:
Bikeshed: maybe instead of `any` it should be `preferNative`? I like `reset` though.

Reply:
I like `reset` as well.